product_docs/docs/efm/4/efm_quick_start/index.mdx

```diff
@@ -30,48 +30,48 @@ Start the configuration process on a primary or standby node. Then, copy the con
-| `auto.allow.hosts`      | On a test cluster, set to `true` to simplify startup; for production usage, consult the user's guide.                            |
-| `stable.nodes.file`     | On a test cluster, set to `true` to simplify startup; for production usage, consult the user's guide.                            |
+   | Property                | Description                                                                                                                      |
+   | ----------------------- | -------------------------------------------------------------------------------------------------------------------------------- |
+   | `db.user`               | The name of the database user.                                                                                                   |
+   | `db.password.encrypted` | The encrypted password of the database user.                                                                                     |
+   | `db.port`               | The port monitored by the database.                                                                                              |
+   | `db.database`           | The name of the database.                                                                                                        |
+   | `db.service.owner`      | The owner of the `data` directory (usually `postgres` or `enterprisedb`). Required only if the database is running as a service. |
+   | `db.service.name`       | The name of the database service (used to restart the server). Required only if the database is running as a service.            |
+   | `db.bin`                | The path to the `bin` directory (used for calls to `pg_ctl`).                                                                    |
+   | `db.data.dir`           | The `data` directory in which EFM will find or create the `recovery.conf` file or the `standby.signal` file.                     |
+   | `user.email`            | An email address at which to receive email notifications (notification text is also in the agent log file).                      |
+   | `bind.address`          | The local address of the node and the port to use for Failover Manager. The format is: `bind.address=1.2.3.4:7800`                            |
+   | `is.witness`            | `true` on a witness node and `false` if it is a primary or standby.                                                              |
+   | `ping.server.ip`        | If you are running on a network without Internet access, set `ping.server.ip` to an address that is available on your network.   |
+   | `auto.allow.hosts`      | On a test cluster, set to `true` to simplify startup; for production usage, consult the user's guide.                            |
+   | `stable.nodes.file`     | On a test cluster, set to `true` to simplify startup; for production usage, consult the user's guide.                            |
```

Comment:
As long as the final one looks ok with these whitespace changes i'm ok with it. I don't know what effect this has on the doc.